cernopendata/templates/cernopendata_theme/footer.html

```diff
@@ -4,65 +4,65 @@
       <div class="six wide column">
         <div class="ui list horizontal logos">
           <div class="item">
-            <a href="http://alice.cern"
-              ><img
+            <a href="http://alice.cern" aria-label="ALICE"
```

Comment:
I think that having the `img` with an `alt` text within the link  is enough, the link itself does not need an `aria-label` since it's redundant.

Reply:
Thanks. I thought that lighthouse recommended adding this aria-label, but I couldn't reproduce it now. So, all of those aria-labels have been removed